sycl/tools/sycl-xocc/bin/sycl-xocc

```diff
@@ -35,13 +35,13 @@
 
 # TODO: Pass an argument through the driver to set options like XILINX_PLATFORM,
 # the environment variables are a quick workaround to swap boards.
-usage() { echo xocc_compile: error: $2 >&2; exit $1; }
+usage() { echo sdx_compile: error: $2 >&2; exit $1; }
```

Comment:
I guess that at this level we do not know yet if it is xocc or v++ to display in the error message...

Reply:
technically you could pass another argument to the script that's basically a true false flag, if you really wanted this error to be specific. But it should probably be sycl-xocc: error: or so in reference to the script. But sdx_compile seems fine as well.